src/helpers/parse_snak.ts

```diff
@@ -105,20 +108,20 @@ for (const [ datatype, parser ] of Object.entries(parsers)) {
   normalizedParsers[normalizeDatatype(datatype)] = parser
 }
 
-export function parseClaim (datatype, datavalue, options, claimId) {
+export function parseSnak (datatype: DataType | void, datavalue: SnakValue, options: SimplifySnakOptions) {
   // Known case of missing datatype: form.claims, sense.claims, mediainfo.statements
+  // @ts-expect-error
```

Comment:
The lint is not yet enabled but in the future the ts-expect-error will require a comment to specify why this error is allowed. So this change basically makes this lint happy.

```suggestion
  // @ts-expect-error Known case of missing datatype: form.claims, sense.claims, mediainfo.statements
```

Reply:
done in bbc2d82